py/desiutil/names.py

```diff
@@ -0,0 +1,62 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+============
+desiutil.names
+============
+
+This package contains functions for naming 'things' in DESI
+or decoding those names
```

Comment:
This is a complete sentence (good!) so it should end with a `.`.

Reply:
Done in latest commits